### PR TITLE
Translation of 2020 http2 chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of #586

### CODE REVIEW

**Comment by ksakae1216** [comment]
@bazzadp Build and Test Website was error.
I confirmed message(**Tag must be paired, no start tag**), but I don't know how to fix.

Please tell me, how to fix.

**Comment by bazzadp** [comment]
> @bazzadp Build and Test Website was error.
> I confirmed message(**Tag must be paired, no start tag**), but I don't know how to fix.
> 
> Please tell me, how to fix.

This is the error it gave:

```
2021-02-06 05:41:03 [ERROR]   Error code: 1. Command output:[
   Config loaded: /github/workspace/.github/linters/.htmlhintrc

   /github/workspace/src/static/html/ja/2020/http2.html
      L1495 |...=&#8221;../2019/http2#fig-5&#8221;&gt;昨年</a<></a
                                                          ^ Tag must be paired, no start tag: [ </a<> ] (tag-pair)
```

Looks to me like you entered `</a>>` instead of `</a>`. However also weird that you have two `</a></a>` tags nright after each other. Links (`<a href="...">`) cannot be nested so you can't have two closing tags next to each other.

File: src/content/ja/2020/http2.md
Comment (bazzadp) [inline src/content/ja/2020/http2.md:142]: This your problem - the link is not in markup format as was split

```suggestion
ウェブ上の[セキュリティ](./security)と[プライバシー](./privacy)を向上させようとする動きにより、TLSを利用したリクエストは[過去4年間で150%以上](https://httparchive.org/reports/state-of-the-web#pctHttps)増加しています。今日では、モバイルとデスクトップにおけるすべてのリクエストの86%以上が暗号化されています。ホームページだけを見ると、デスクトップでは78.1%、モバイルでは74.7%という印象的な数字になっています。これは、HTTP/2がTLSを介したブラウザでのみサポートされていることが重要です。図22.5に示すように、HTTP/2でサービスされる割合も[昨年](../2019/http2#fig-5)から10ポイント増えており、55%から65%になっています。
```

File: src/content/ja/2020/http2.md
Comment (bazzadp) [inline src/content/ja/2020/http2.md:591]:
```suggestion
クライアントとサーバがHTTP/1.1とHTTP/2の両方をサポートしているとしましょう。どちらを使うかはどうやって選ぶのでしょうか？　最も一般的な方法はTLS[Application Layer Protocol Negotiation](https://ja.wikipedia.org/wiki/Application-Layer_Protocol_Negotiation) (ALPN) で、クライアントはサポートするプロトコルのリストをサーバに送信し、サーバはその接続に使用するプロトコルを選択します。 ブラウザはHTTPS接続を設定する際にTLSパラメーターをネゴシエートする必要があるため、同時にHTTPバージョンもネゴシエートできます。HTTP/2とHTTP/1.1の両方が同じTCPポート（443）から提供されるので、ブラウザは接続を開く前にこの選択をする必要はありません。
```

File: src/content/ja/2020/http2.md
Comment (bazzadp) [inline src/content/ja/2020/http2.md:636]:
```suggestion
インターネット上に新しいトランスポートプロトコルを展開しようとする過去の試みは、[Stream Control Transmission Protocol](https://ja.wikipedia.org/wiki/Stream_Control_Transmission_Protocol) (SCTP)のように困難であることが証明されています。QUICはUDPの上で動作する新しいトランスポートプロトコルです。QUICは、信頼性の高い順番配信やネットワークへのフラッディングを防ぐ輻輳制御など、TCPと同様の機能を提供します。
```

File: src/content/ja/2020/http2.md
Comment (bazzadp) [inline src/content/ja/2020/http2.md:599]:
```suggestion
`Upgrade`の最も一般的なアプリケーションは[WebSockets](https://developer.mozilla.org/ja/docs/Web/API/WebSockets_API)です。HTTP/2もまた、暗号化されていないクリアテキストモードで使用するための`Upgrade`パスを定義しています。しかし、Webブラウザでこの機能はサポートされていません。したがって、我々のデータセットに含まれるHTTP/1.1のクリアテキストリクエストのうち、応答に`Upgrade`ヘッダーが含まれていたのは3%未満であったことは驚くに値しません。非常に少数のTLSを使用するリクエスト（HTTP/2の0.0011%、HTTP/1.1の0.064%）もレスポンスで`Upgrade`ヘッダーを受け取っていました。これらは、HTTP/2を話したりTLSを終了させたりしているが、`Upgrade`ヘッダーを適切に削除していない仲介者の背後にあるクリアテキストHTTP/1.1サーバーである可能性が高いです。
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
LGTM once we merge these few minor tweaks.

**Review by bazzadp — APPROVED** [review]

**Comment by bazzadp** [comment]
Going to merge this as didn't get any feedback on translation itself last time. Would love reviews if anyone gets a chance through so let me know and will hold off merging next time to give a chance to review.

@ksakae1216 noticed I got an email alter when committing those last changes. IF you rename your `master` branch to `main` (or alternatively just delete your fork, and re-fork it) then you won't get that with each commit in future.

**Pull request merged**
